src/GDGUkraine/controller.py

```diff
@@ -42,6 +42,7 @@ def index(self, **kwargs):
                                           filtered=True))
 
     @cherrypy.expose
+    @cherrypy.tools.authorize()
```

Comment:
1. That page should show login buttons, so we have to show it.
2. Nevertheless this is useless handler, nobody will reach it, since it's being overridden by static handler in config.